go-controller/pkg/ovn/master.go

```diff
@@ -648,7 +649,12 @@ func (oc *Controller) addNodeAnnotations(node *kapi.Node, hostSubnets []*net.IPN
 		return fmt.Errorf("failed to marshal node %q annotation for subnet %s",
 			node.Name, util.JoinIPNets(hostSubnets, ","))
 	}
-	err = oc.kube.SetAnnotationsOnNode(node, nodeAnnotations)
+	err = utilwait.PollImmediate(100*time.Millisecond, 1*time.Second,
+		func() (bool, error) {
+			err = oc.kube.SetAnnotationsOnNode(node, nodeAnnotations)
```

Comment:
I guess it would be nice to log an info message depicting the error and suggesting we are retrying.

Reply:
ack